src/Output.cxx

```diff
@@ -732,10 +741,6 @@ ASTVisitor::DumpId ASTVisitor::AddTypeDumpNode(DumpType dt, bool complete,
       return this->AddTypeDumpNode(
         DumpType(t->getAs<clang::DecayedType>()->getDecayedType(), c),
         complete, dq);
-    case clang::Type::Elaborated:
-      return this->AddTypeDumpNode(
-        DumpType(t->getAs<clang::ElaboratedType>()->getNamedType(), c),
-        complete, dq);
```

Comment:
We should be able to keep the logic in this `case` and just `break` when it is not gccxml mode.